docs/pages/_app.js

```diff
@@ -248,7 +247,6 @@ export default function MyApp(props) {
 
   return (
     <AppWrapper emotionCache={emotionCache} pageProps={pageProps}>
-      <BackToTop />
```

Comment:
Move the component to only be visible on the docs pages, having it in https://mui.com/material-ui/getting-started/templates/blog/ feels deceptive, it's not what the template includes